db/seeds.rb

```diff
@@ -9,8 +9,8 @@
e and destroys his bedroom, Donnie is not there. Both the event, and Donnie's escape, seem to have been caused by supernatural events. Donnie's mental illness, if such it is, may never allow him to find out for sure.", small_cover: '/tmp/donnie_darko_small.jpg', large_cover: '/tmp/donnie_darko_large.jpg', category_id: 4}, {title: 'Dances With Wolves', description: "Lt. John Dunbar is dubbed a hero after he accidentally leads Union troops to a victory during the Civil War. He requests a position on the western frontier, but finds it deserted. He soon finds out he is not alone, but meets a wolf he dubs 'Two-socks' and a curious Indian tribe. Dunbar quickly makes friends with the tribe, and discovers a white woman who was raised by the Indians. He gradually earns the respect of these native people, and sheds his white-man's ways.", small_cover: '/tmp/dances_wolves_small.jpg', large_cover: '/tmp/dances_wolves_large.jpg', category_id: 1}, {title: 'The Jerk', description: "Navin is an idiot. He grew up in Mississippi as adopted son of a black family but on his 18th birthday he feels he wants to discover the rest of the world and sets out to St. Louis. There everyone exploits his naivety, but then a simple invention brings him a fortune", small_cover: '/tmp/the_jerk_small.jpg', large_cover: '/tmp/the_jerk_large.jpg', category_id: 2}, {title: 'Once Upon A Time In The West', description: "Story of a young woman, Mrs. McBain, who moves from New Orleans to frontier Utah, on the very edge of the American West. She arrives to find her new husband and family slaughtered, but by whom? The prime suspect, coffee-lover Cheyenne, befriends her and offers to go after the real killer, assassin gang leader Frank, in her honor. He is accompanied by Harmonica on his quest to get even.", small_cover: '/tmp/once_upon_small.jpg', large_cover: '/tmp/once_upon_large.jpg', category_id: 1}, {title: 'Unbreakable', description: "Security Guard David Dunn miraculously survives a catastrophic train crash outside Philadelphia. Not only is he the sole survivor out of 132 passengers, he also is completely unharmed. A little later, comic book specialist Elijah Price contacts him to confront David with an incredible theory: Elijah, who has been nicknamed 'Mr. Glass' due to his more than fragile bones, thinks that David has got all which he himself lacks. The two of them 'seem to be linked by a curve, but sitting on opposite ends'. First, David does not believe the strange man, but every single thing he had said proves to be true: David has never ever been hurt or sick in his life, his physical strength is larger than normal and he has a skill which others don't. Slowly, David begins to discover the shocking truth behind Mr. Price's assumptions. But after all, David's fate is not only to find his real place in the world. It also is about proving Elijah's theory of his own existence.", small_cover: '/tmp/unbreakable_small.jpg', large_cover: '/tmp/unbreakable_large.jpg', category_id: 3}])
```

Comment:
Also I would recommend using `create!` - it throws an exception when there is a validation error while seeding. Now, it silently ignores all errors.